openapi.yaml

```diff
@@ -1346,13 +1355,17 @@ paths:
         | ------------|:----------------------------:|
         | MWA         | The Market weighted averages |
         | GWA         | The Global weighted averages |
+        | LX          | Liquid Indices               |
 
         Each index type will have an index id which identifies the constituent type used to build the index -
 
         | Index Type | Index Id  | Output |
         | -----------|:---------:|:---------:|
         | MWA        | Market Id | The Market Weighted Average for that market |
         | GWA        | Asset Id  | The Global Weighted Average for that coin / asset |
+        | LX         | Index Id  | Bitcoin Liquid Index (BLX)
+        | LX         | Index Id  | Ethereum Liquid Index (ELX)
+        | LX         | Index Id  | Ripple Liquid Index (XRPLX)
```

Comment:
We can use the actual UUIDs for the BLX, ELX, XRPLX here. See https://docs.bravenewcoin.com/#tag/Websocket-LX though that uses an `Example Id`

Also missing end column.